cmd/download.go

```diff
@@ -5,9 +5,15 @@ package cmd
 
 import (
 	"github.com/cisco-open/grabit/internal"
+	"github.com/rs/zerolog"
 	"github.com/spf13/cobra"
 )
 
+// This Go code defines a command-line interface (CLI) command for downloading resources.
```

Comment:
I like the idea of documenting methods... but:
- Please do it in another PR so that this one can be focused on one thing and one thing only
- Follow the standard guidelines for documenting go code https://go.dev/blog/godoc